cpp/src/Ice/UdpEndpointI.cpp

```diff
@@ -12,6 +13,7 @@
 #include "ProtocolInstance.h"
 #include "UdpConnector.h"
 #include "UdpTransceiver.h"
+#include <utility>
```

Comment:
I think in most sources we have a new lien between Ice and other headers, I guess the clang-tidy added those and doesn't know this rule.

Reply:
I removed most of the new `include <utility>`